pkg/web/templates/components/add_apikey.html

```diff
@@ -0,0 +1,130 @@
+<dialog id="add_apikey_modal" class="modal">
+  <div class="modal-box max-w-2xl">
+    <div class="flex justify-between items-center">
+      <h1 class="font-bold text-xl">Create API Key</h1>
+      <button onclick="add_apikey_modal.close()" class="btn btn-sm btn-circle btn-ghost">
+        <i class="fa-solid fa-x"></i>
+        <span class="sr-only">Close modal</span>
+      </button>
+    </div>
+    <p class="py-4">Name your key and select access permissions.</p>
+    <div>
+      <form
+        id="new-apikey-form"
+        hx-post="/v1/apikeys" 
+        hx-ext="json-enc" 
+        hx-target="#apikey_modal" 
+        hx-swap="outerHTML"
+        hx-indicator="#loader" 
+        method="post"
+        >
+        <div class="mb-4">
+          <label for="description" class="label-style">API Key Description</label>
+          <input type="text" id="description" name="description" class="input-style" />
+        </div>
+        <h2 class="font-bold">Permissions:</h2>
+        <section class="p-2 my-2 border border-gray-300 rounded">
+          <h3 class="font-bold">Full Access</h3>
+          <div class="flex justify-between">
+            <label for="full_access" class="max-w-4xl text-balance">(Manage accounts, View accounts, Manage api keys, View api keys, Revoke api keys, Manage config, View config, Manage counterparties, View counterparties, 
+              Manage pki, View pki, Delete pki, Manage transactions, View transactions, Manage travel rule, View travel rule, Delete travel rule, Manage users, View users)
+            </label>
+            <div class="pt-1">
+              <input type="checkbox" checked id="full_access" name="permissions" value="full" class="checkbox-style" />
+            </div>
+          </div>
+        </section>
+        <section class="p-2 my-4 border border-gray-300 rounded">
+          <div class="flex justify-between">
+            <label for="custom_access" class="font-bold">Custom Access</label>
+            <div class="pt-1">
+              <input type="checkbox" id="custom_access" class="checkbox-style" />
```

Comment:
Do we want to hide the permissions and then display them once a users clicks custom access? If yes, the custom permissions will be hidden and once the user clicks this checkbox they will appear.

Reply:
No, I like having the permissions listed by default, so long as the vertical scroll doesn't hide the submit button (e.g. the permissions list should scroll independently of the rest of the modal).

Reply:
I'll make the update while working on the delete modal view.